### ISSUE TITLE
[webview_flutter] The video which is playing that will be paused, After screen orientation was changed.

### ISSUE THREAD

**Issue by windows7lake** [flutter/flutter#51915]
Playing a video in webview and then change screen's orientation, the webview seem to be repaint and video was paused.
What can I do to prevent the video from being paused？

**Comment by windows7lake** [comment]
I am using webview_flutter example and tested on it. But I didn't find any way to deal with it.

**Comment by deandreamatias** [comment]
Hi,
can you provide your `flutter doctor -v` and some code to test this please ?

**Comment by windows7lake** [comment]
flutter doctor --verbose
[✓] Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.15.3 19D76, locale zh-Hans-CN)
    • Flutter version 1.12.13+hotfix.8 at /Users/yinglin/Library/flutter
    • Framework revision 0b8abb4724 (3 weeks ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/yinglin/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/yinglin/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] Connected device (1 available)
    • Pixel 3 XL • 89SY0B6DE • android-arm64 • Android 10 (API 29)

• No issues found!
Process finished with exit code 0

**Comment by windows7lake** [comment]
@deandreamatias  Download [webview_flutter] in flutter plugin, these is a example in it.(It's the same as my example.) You can run the example and then click the video on home page to playing. Everytime we change orientation of devices, the video will be paused.

**Comment by VladyslavBondarenko** [comment]
Checked with real Android 10, indeed it pauses, while with Android 10 simulator it doesn't

**Comment by windows7lake** [comment]
checked with real devices, no only android 10, but also on the other android version the video will be paused.

**Comment by windows7lake** [comment]
Who can offer some feedback? Please!

**Comment by cg021** [comment]
This seems to be Youtube specific. I haven't been able to reproduce with videos from other sites (e.g. Vimeo).

**Comment by darshankawar** [comment]
Issue reproducible on latest `dev channel (1.20.0-0.0.pre)` and with latest version of `webview_flutter`. 

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel dev, 1.20.0-0.0.pre, on Mac OS X 10.15.2 19C57, locale en-US)
    • Flutter version 1.20.0-0.0.pre at /Users/deeptibelsare/Documents/Fluttersdk/flutter
    • Framework revision d9653445f4 (8 days ago), 2020-06-09 18:43:03 -0400
    • Engine revision e8c13aa012
    • Dart version 2.9.0 (build 2.9.0-14.0.dev 5c1376615e)

 
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/deeptibelsare/Library/Android/sdk
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /Users/deeptibelsare/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/deeptibelsare/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    ✗ Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions.

 
[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.9.3

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 8.1.0 (API 27) (emulator)

! Doctor found issues in 1 category.
```

</details>

**Comment by blasten** [comment]
This has been fixed in Flutter 1.22/ WebView 1.0 - use hybrid composition in the Flutter WebView plugin.

https://github.com/flutter/plugins/tree/master/packages/webview_flutter#using-hybrid-composition
